projects/packages/videopress/src/utility-functions.php

```diff
@@ -424,7 +424,7 @@ function videopress_is_attachment_without_guid( $post_id ) {
 function is_videopress_attachment( $post_id ) {
 	$post = get_post( $post_id );
 
-	if ( is_wp_error( $post ) ) {
+	if ( ! is_a( $post, 'WP_Post' ) ) {
```

Comment:
Why not `instanceof`?
```suggestion
	if ( ! $post instanceof WP_Post ) {
```

Reply:
`null instanceof WP_Post` gives a fatal in PHP 7.2:
https://php-errors.readthedocs.io/en/latest/messages/instanceof-expects-an-object-instance%2C-constant-given.html

Reply:
Sure, `null instanceof WP_Post` gives a fatal in 7.2. But `$var instanceof WP_Post` is fine even if `$var === null`.
https://3v4l.org/vFWs9

In other words, the error is about using a literal value instead of a variable on the left hand side..

Reply:
Heh, at least I tested, but apparently I tested the wrong way. 😓